src/index.js

```diff
@@ -0,0 +1,192 @@
+
+const serverAddress = 'http://127.0.0.1:5000';
+
+const populateWeatherReport = function (weatherData) {
+  const tempKelvin = weatherData.current.temp;
+  const tempFahrenheit = Math.floor((tempKelvin - 273.15) * 1.8 + 32);
+  state.temp = tempFahrenheit;
+  colorTempChange();
+  chooseSky(weatherData.current.weather[0].id);
+};
+
+const chooseSky = (skyData) => {
+  let sky = '';
+  if (
+    (skyData >= 701 && skyData <= 781) ||
+    (skyData >= 803 && skyData <= 804)
+  ) {
+    sky = 'Cloudy';
+  } else if (skyData >= 200 && skyData <= 599) {
+    sky = 'Rainy';
+  } else if (skyData >= 600 && skyData <= 700) {
+    sky = 'Snowy';
+  } else if (skyData >= 800) {
+    sky = 'Sunny';
+  }
+  let skySelect = document.querySelector('#skySelect');
+  skySelect.value = sky;
+  changeSky();
+};
+
+const cityCallWeather = function (serverAddress, city) {
+  axios
+    .get(`${serverAddress}/location`, {
+      params: {
+        q: city,
+      },
+    })
+    .then((response) => {
+      if (response.data.error === 'Unable to geocode') {
+        throw 'Invalid city name';
+      }
+
+      let longitude = response.data[0].lon;
+      let latitude = response.data[0].lat;
+
+      axios
+        .get(`${serverAddress}/weather`, {
+          params: {
+            lat: latitude,
+            lon: longitude,
+          },
+        })
+        .then((response) => {
+          populateWeatherReport(response.data);
+        })
+        .catch((response) => {
+          console.log(response.status);
+          console.log(
+            'There was an unexpected issue with the weather API request.'
+          );
+        });
+    })
+    .catch((response) => {
+      const cityName = document.querySelector('#cityName');
+      cityName.textContent = 'Invalid City Name';
+    });
+};
+
+//////////*    CITY NAME ENTRY    *////////////////////////////////////////////
+
+const inputElement = document.querySelector('#userInput');
+
+const resetInput = () => {
+  const cityName = document.querySelector('#cityName');
+  inputElement.value = '';
+  cityName.textContent = 'Seattle';
+  cityCallWeather(serverAddress, cityName.textContent);
+};
+
+const changeCityName = (event) => {
+  const cityName = document.querySelector('#cityName');
+  const inputCityName = event.target.value;
+  if (inputCityName === '') {
+    cityName.textContent = 'Seattle';
+  } else {
+    normalizeCityName(inputCityName);
+  }
+  cityCallWeather(serverAddress, cityName.textContent);
+};
+
+normalizeCityName = (inputCityName) => {
+  const normalizedCityName = inputCityName
+    .toLowerCase()
+    .split(' ')
+    .map((s) => s.charAt(0).toUpperCase() + s.substring(1))
+    .join(' ');
+  cityName.textContent = normalizedCityName;
+};
+
+const getRealTemp = () => {
+  const cityName = document.querySelector('#cityName');
+  cityCallWeather(serverAddress, cityName.textContent);
+};
+
+//////////*    TEMPERATURE DIFFERENCE    */////////////////////////////////////
+('use strict');
```

Comment:
It's good that you attempted to invoke strict mode but this isn't one of the two options for doing so: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode